app/models/spotlight/contact_form.rb

```diff
@@ -4,10 +4,15 @@ module Spotlight
   class ContactForm
     include ActiveModel::Model
 
-    attr_accessor :current_exhibit, :name, :email, :message, :current_url, :request
+    attr_accessor :current_exhibit, :name, :email, :email_address, :message, :current_url, :request
 
     validates :email, format: { with: /\A([\w\.%\+\-]+)@([\w\-]+\.)+([\w]{2,})\z/i }
 
+    # the email_address field is intended to be hidden visually from the user, in hope that
+    # a spam bot filling out the form will enter a value, whereas a human with a browser wouldn't,
+    # allowing us to differentiate and reject likely spam messages.
+    validates :email_address, length: { is: 0 }
```

Comment:
Do we know that we need to call it something that sounds vaguely useful, instead of something that developers can read as bogus data?